fix: Update Get Involved cards

GeoBruit opened the pull request:
<!--- This template can be modified slightly to the needs of the pull request -->

## Description

I have changed the text inside the Get Involved card (as advice on issue #1971 )

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
-Accessed the get involved page (on the local host -> /getinvolved/)
- Checked  the card to make sure the text  was updated
- Check if the read more button leads to the correct URL
- The "Read More" button was already redirecting to the correct page (no changes needed) -> /codingClub/
## Checklist:

Acceptance Criteria
- [x] Button linked to clubs page (Already implemented)
- [x] Text updated

Copywrite Requirements
- [x] Update text to say: Starting a coding club of your own
- [x] Update text to say: Become a Code for Life ambassador by starting up a club and sharing your experience with us.
- [x] Button text: Read More (Already implemented)

<!-- Reviewable:start -->
- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ocadotechnology/codeforlife-portal/2012)
<!-- Reviewable:end -->

KamilPawel reviewed (changes requested):
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/ocadotechnology/codeforlife-portal/2012)* status: all files reviewed, 1 unresolved discussion (waiting on @GeoBruit)

___
*[`portal/templates/portal/getinvolved.html` line 20 at r1](https://reviewable.io/reviews/ocadotechnology/codeforlife-portal/2012#-NM9GXJV7qBsTcslKoBW:-NM9GXJV7qBsTcslKoBX:b-uy0h7d) ([raw file](https://github.com/ocadotechnology/codeforlife-portal/blob/1626ff91dfb473fd8c4dafab1b1279a03ed940c5/portal/templates/portal/getinvolved.html#L20)):*
> ```HTML
>                 <div>
>                     <h4 class="card__title">Starting a coding club</h4>
>                     <p>Become a Code for Life ambassador by starting up a coding club. Find out more about
> ```

We had a slight change in the design, could you leave the last sentence "Find out more about
                    how you can get involved with this by visiting our coding club page." in please :)



_Code quote:_
```HTML
                    <p>Become a Code for Life ambassador by starting up a coding club. Find out more about
                    how you can get involved with this by visiting our coding club page.
```


<!-- Sent from Reviewable.io -->

GeoBruit commented:
Sure!! I ll do the changes as soon As possible and commit again!
I am at work so it might take a few hrs.
Thank you!

GeoBruit commented:
Hello, The changes have been made!
Thank you!

KamilPawel reviewed (changes requested):
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/ocadotechnology/codeforlife-portal/2012)* status: all files reviewed, 1 unresolved discussion (waiting on @GeoBruit)

___
*[`portal/templates/portal/getinvolved.html` line 20 at r1](https://reviewable.io/reviews/ocadotechnology/codeforlife-portal/2012#-NM9GXJV7qBsTcslKoBW:-NMkiKb-2jd_mJuUO81Z:b-n5m6j) ([raw file](https://github.com/ocadotechnology/codeforlife-portal/blob/1626ff91dfb473fd8c4dafab1b1279a03ed940c5/portal/templates/portal/getinvolved.html#L20)):*
<details><summary><i>Previously, KamilPawel wrote…</i></summary><blockquote>

We had a slight change in the design, could you leave the last sentence "Find out more about
                    how you can get involved with this by visiting our coding club page." in please :)
</blockquote></details>

Sorry to keep requesting for more :sweat_smile: but could you also align the button on that card. Also if you could please add a "fix: " prefix to your PR title as the semantic PR check seems to be failing 


<!-- Sent from Reviewable.io -->

GeoBruit commented:
Hi, I added the prefix, sure I ll try to align the button!😌

GeoBruit commented:
Hello, I have aligned the "Read more" button! :)

GeoBruit commented:
Hi @KamilPawel, Sure thing I ll do it right away!

GeoBruit commented:
hi, @KamilPawel I have made the changes as requested, however, I did not manage to make them effective on the front end part.
Please let me know if there s anything I could do or how to do it. Thank you!

KamilPawel reviewed (changes requested):
> *[Reviewable](https://reviewable.io/reviews/ocadotechnology/codeforlife-portal/2012)* status: 0 of 2 files reviewed, 3 unresolved discussions (waiting on @GeoBruit)

___
*[`portal/static/portal/sass/partials/_grids.scss` line 87 at r4](https://reviewable.io/reviews/ocadotechnology/codeforlife-portal/2012#-NNlVlSO1lSIaEpFkQOF:-NNlVlSO1lSIaEpFkQOG:bhm3tm9) ([raw file](https://github.com/ocadotechnology/codeforlife-portal/blob/dbfdac923624021c38481d76f53f2f3af6f10c4c/portal/static/portal/sass/partials/_grids.scss#L87)):*
> ```SCSS
>     flex-direction: column;
>     flex-grow: 1;
>     justify-content: space-between;
> ```

I believe you placed this line in a statically generated file, which are in .gitignore. There is another path with the exact file type and name that you need to paste this line in :)


<!-- Sent from Reviewable.io -->

GeoBruit commented:
Hi @KamilPawel, I have found the other file at code for life-portal\static\portal\sass\partials\_grids.scss, however, the line of code was already there, So I just removed it from where I added it previously,
    display: flex;
    flex-direction: column;
    flex-grow: 1;
    justify-content: space-between;

I have checked on the local host and the button still doesn't seem to be aligned, please let me know what else could I do :-)

KamilPawel reviewed (changes requested):
> *[Reviewable](https://reviewable.io/reviews/ocadotechnology/codeforlife-portal/2012)* status: 0 of 2 files reviewed, 3 unresolved discussions (waiting on @GeoBruit)

___
*[`portal/static/portal/sass/partials/_grids.scss` line 87 at r4](https://reviewable.io/reviews/ocadotechnology/codeforlife-portal/2012#-NNlVlSO1lSIaEpFkQOF:-NOOe9eh6mJTGEZH_cxO:bugx83g) ([raw file](https://github.com/ocadotechnology/codeforlife-portal/blob/ce16b0edbd5ed984796c2e7dd85aa5bcaaff2df9/portal/static/portal/sass/partials/_grids.scss#L87)):*
<details><summary><i>Previously, KamilPawel wrote…</i></summary><blockquote>

I believe you placed this line in a statically generated file, which are in .gitignore. There is another path with the exact file type and name that you need to paste this line in :)
</blockquote></details>

Can you try adding the instance of the .card__text with justify-content: space-between in _buttons.scss in directory portal/static/portal/sass/partials? Once the change is applied, restart the server and clear your cache by performing hard refresh. This should work :)


<!-- Sent from Reviewable.io -->

GeoBruit commented:
@KamilPawel I will give it a try, thank you!:)

KamilPawel commented:
> @KamilPawel I will give it a try, thank you!:)

Hi GeoBruit, we will be taking over the task. Thank you for your contribution :)

faucomte97 reviewed (approved):
> Reviewed all commit messages.
> Dismissed @KamilPawel from a discussion.
> *[Reviewable](https://reviewable.io/reviews/ocadotechnology/codeforlife-portal/2012)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @GeoBruit)



<!-- Sent from Reviewable.io -->
